shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/repository/ResetPasswordRepository.kt

```diff
@@ -0,0 +1,9 @@
+package co.nimblehq.blisskmmic.domain.repository
+
+import co.nimblehq.blisskmmic.data.model.ResetPasswordMeta
+import kotlinx.coroutines.flow.Flow
+
+interface ResetPasswordRepository {
+
+    fun reset(email: String): Flow<ResetPasswordMeta>
+}
```

Comment:
This looks like we can make use of the same generic name strategy for this repository:
```suggestion
interface AccountRecoveryRepository {

    fun resetPasswordWith(email: String): Flow<ResetPasswordMeta>
}
```